### DISCUSSION TITLE
Wish Copilot was included with Pro membership

### DISCUSSION THREAD

**Discussion by robd003** [community/community#19248] [Copilot Conversations] [answered]
I really can't justify spending an extra $10/month for Copilot since the quality just isn't there. (I was using Copilot as a beta tester)

I would use it if it was included as part of my GitHub Pro membership.

**Comment by melvincwng** [comment, marked as answer]
Hi @robd003, are you a student? If yes, Github Copilot is free to use for students!

Otherwise, for majority of users, there's a free 60 day trial for GitHub Copilot.

Hope this clarifies.

Source: [Github Blog](https://github.blog/2022-06-21-github-copilot-is-generally-available-to-all-developers/)
